assets/vue/components/BlockedIncident.vue

```diff
@@ -25,6 +25,7 @@
 </template>
 
 <script>
+import * as Filtering from '../../filtering.js';
```

Comment:
Helper functions in vue apps go into a subfolder, like `assets/vue/helpers/filtering.js`.